scripts/jenkins/cleanvm/openstack-cleanvm.builder.sh

```diff
@@ -29,7 +29,9 @@ set +e
 scp ${automationrepo}/scripts/jenkins/qa_openstack.sh root@cleanvm:
 ssh root@cleanvm "export cloudsource=$cloudsource; export OSHEAD=$oshead; export NONINTERACTIVE=1; bash -x ~/qa_openstack.sh"
 ret=$?
+echo "Exit code of cleanvm run: $ret"
 if [ "$ret" != 0 ] ; then
+  echo "The cleanvm run failed. Now trying to cleanup before we let this job fail."
   virsh shutdown cleanvm
```

Comment:
output can be redirected to /dev/null because we are fine with missing VMs that cannot be cleaned up